web-app/src/App.js

```diff
@@ -3,15 +3,21 @@ import './App.css';
 import { Switch, Route } from 'react-router-dom';
 import Homepage from './homepage';
 import Chatpage from './chatpage';
+import SignInPage from './signinPage'
 import Bulletinpage from './bulletinpage';
+import Header from './header';
 
 function App() {
   return (
-    <Switch>
-      <Route exact path='/' component={Homepage}></Route>
-      <Route exact path='/bulletin' component={Bulletinpage}></Route>
-      <Route exact path='/chat' component={Chatpage}></Route>
-    </Switch>
+    <div>
+      <Header/>
```

Comment:
Why did this need a div & header just to add signin?

Reply:
The `Header` is the new component from `header.js` that represents the overall app header that we had put up throughout the app, I just abstracted it to it's own component
![image](https://user-images.githubusercontent.com/15304203/80774568-a745c700-8b1a-11ea-91a2-6cb2c35ed3e6.png)